CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.2 (November 17, 2023). Tested on Artifactory 7.71.4 and Xray 3.85.5
```

Comment:
Do you want to bump a version?

Reply:
I literally just noticed it a few minutes ago. I've already pushed an update.